src/renderers/shared/stack/event/EventPluginRegistry.js

```diff
@@ -236,7 +260,9 @@ var EventPluginRegistry = {
    * @return {?object} The plugin that created the supplied event.
    * @internal
    */
-  getPluginModuleForEvent: function(event) {
+  getPluginModuleForEvent: function(
+    event: ReactSyntheticEvent
+  ): null | PluginModule {
```

Comment:
`null | Type` is how @sebmarkbage has been writing it too. I think it sort of follows from the use of `?Type`. It's also more readable in cases where you might have a function (eg [here](https://github.com/facebook/react/blob/4c365ea9573f2f4b0af6ddebc9397fc778b65712/src/renderers/shared/fiber/ReactFiber.js#L73))

I agree that it feels more natural to have `null` as the last thing since it seems least important but I'm on board with the pattern we have here.